app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+use Hash;
+
+class LoginController extends Controller
+{
+    public function Login(Request $request)
+    {
+        $credentials = $request -> validate ([
+            'email' => ['required', 'email'],
+            'password' => ['required'], ]);
```

Comment:
I would move that into `FormRequest` - https://laravel.com/docs/10.x/validation#form-request-validation

Reply:
done